code/__DEFINES/monitor.dm

```diff
@@ -29,12 +29,12 @@
 #define HUMAN_LIFE_ON_SHIP_WEIGHT 70
 #define BURROWED_LARVA_WEIGHT -20
 #define REQ_POINTS_WEIGHT 0.2
-#define ELDER_T2_WEIGHT -40
-#define ANCIENT_T2_WEIGHT -60
-#define ELDER_T3_WEIGHT -60
-#define ANCIENT_T3_WEIGHT -100
-#define ELDER_T4_WEIGHT -80
-#define ANCIENT_T4_WEIGHT -120
+#define NORMAL_T2_WEIGHT -40
```

Comment:
Ehh does primo really justify that much more weight? I guess it depends on the caste. Bleh.

Reply:
tbh its the old elder/ancient jump